homeassistant/components/deconz/siren.py

```diff
@@ -63,17 +77,17 @@ def __init__(self, device, gateway) -> None:
         )
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if siren is on."""
-        return self._device.is_on
+        return bool(self._device.is_on)
```

Comment:
```suggestion
        return self._device.is_on  # type: ignore[no-any-return]
```
If you don't have to cast a `None` value.